CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.90.0
+
+### Breaking changes
+
+* @PrimaryKey field value can be null for java.lang.String type, and should be migrated by using io.realm.internal.Table.convertColumnToNullable() for older version of Realm (#2515).
```

Comment:
`Table.convertColumnToNullable` is a non-public API. They should use `RealmObjectSchema.setNullable(true)` instead.